tmt-app/src/Components/Quiz.jsx

```diff
@@ -1,13 +1,25 @@
 import React from "react";
-import { Container, Grid } from "@mui/material";
+import { useParams } from "react-router-dom";
+import { Container, Grid, Typography } from "@mui/material";
+import { CustomTheme } from "../Styling/CustomStyling";
 import QuestionCard from "./QuestionCard";
-import { CAREERS_CARDS_PLACEHOLDERS } from "../constants/careerMajorList";
+import BobaBot from "./BobaBot";
+import {
+  CAREERS_CARDS_PLACEHOLDERS,
+  SOFT_SKILL_LIST,
+} from "../constants/careerMajorList";
 
 function Quiz() {
+  const { step } = useParams();
+
+  const datasets = { 1: CAREERS_CARDS_PLACEHOLDERS, 2: SOFT_SKILL_LIST };
+  const data = datasets[step] || datasets[1];
```

Comment:
should this be `datasets[1]` or `datasets[0]`?

also while we're here, we should use camelCasing for variable names

Reply:
I think it represents the current step or defaults to 1 if out of bounds